torchao/experimental/ops/mps/setup.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
why this file at all?

Reply:
This allows me to do:
`pip install .`
in order to install the custom library